python/src/opendp/_convert.py

```diff
@@ -514,6 +517,11 @@ def _hashmap_to_slice(val: dict[Any, Any], type_name: RuntimeType) -> FfiSlicePt
     ffislice.depends_on(keys, vals)
     return ffislice
 
+def _slice_to_function(raw: FfiSlicePtr) -> dict[Any, Any]:
+    from opendp.mod import Function
+    function = ctypes.cast(raw.contents.ptr, ctypes.POINTER(AnyFunction)).contents
+    # put the contents behind a new, python pointer
+    return ctypes.cast(ctypes.pointer(function), Function)
```

Comment:
Hopefully mypy is checking this, but I'm surprised that the return type is `dict`: I think it would return an instance of `Function`?